go.mod

```diff
@@ -7,18 +7,20 @@ require (
 	github.com/anmitsu/go-shlex v0.0.0-20161002113705-648efa622239 // indirect
 	github.com/apenwarr/fixconsole v0.0.0-20191012055117-5a9f6489cc29
 	github.com/coreos/go-iptables v0.4.5
+	github.com/coreos/go-systemd/v22 v22.1.0 // indirect
```

Comment:
My package doesn't depend on this; did you `go mod tidy`?

Reply:
That must be left over from my first experiment. I thought go modules automatically cleaned up for you. I'll fix this in the morning.